api/src/services/programsService.ts

```diff
@@ -9,18 +9,40 @@ import {
 import { DateTime, Duration } from 'luxon';
 
 /**
- * Returns the programs associated with an optionally specified curriculum ID.
- * If ID is not specified, returns all programs.
+ * Returns all programs in the database.
  *
- * @param {number} curriculumId - optional ID for a specified curriculum
- * @returns {Program[]} - all matching programs, or all programs
+ * @returns {Program[]} - all programs, or null if not found
  */
-export const listPrograms = (curriculumId?: number): Promise<Program[]> => {
-  if (curriculumId) {
-    return db<Program>('programs').where({ curriculum_id: curriculumId });
-  } else {
-    return db<Program>('programs');
-  }
+export const listAllPrograms = async () => {
```

Comment:
Good job with this function. Great work changing from returning a specific program associated with a curriculum ID to returning all programs.